benchmark_runner/common/oc/oc.py

```diff
@@ -280,7 +280,7 @@ def apply_security_privileged(self, namespace: str = environment_variables.envir
     def create_async(self, yaml: str, is_check: bool = False):
         """
         This method create yaml in async
-        :param yaml:
+        @param yaml:
```

Comment:
What is the purpose of this?

Reply:
its syntax's mistake, parameter in comment should start with @ and not :

Reply:
OK, I guess this will eventually get fixed elsewhere.